django_x509/static/django-x509/js/switcher.js

```diff
@@ -0,0 +1,42 @@
+(function($){
+    // function for operation_type switcher
+    var filter_div = function(operation_type){
+    // define fields for each operation_type
+    var import_fields = $('.form-row:not(.field-certificate, .field-operation_type, .field-private_key)')
```

Comment:
use only one var statement at the beginning of the function and separate them by `,`, the last statement will need a `;`